### ISSUE TITLE
liga-hiszpanska.footballclub.pl

### ISSUE THREAD

**Issue by felix-22** [MajkiIT/polish-ads-filter#19559]
`cookies / rodo`

![ss_182219_liga-hiszpanska footballclub pl_](https://user-images.githubusercontent.com/75935068/114261004-729fe800-99d8-11eb-9faa-f7e1c0b7ac3b.png)

**Comment by strict-block** [comment]
2 będą jak zlikwidują redirect

**Comment by felix-22** [comment]
No, bot nie informuje, czy był redirect czy nie, i nie chce mi się tego implementować, a ja sam też tego nie sprawdzam.
